XenModel/Actions/VMPP/CreateVMPP.cs

```diff
@@ -37,47 +37,38 @@
 
 namespace XenAdmin.Actions
 {
-    public class CreateVMPolicy<T> : AsyncAction where T : XenObject<T>
+    public class CreateVMPolicy : AsyncAction

```

Comment:
You may want to rename this file to match the action's name. (The containin folder name is not relebant either, but I'm not insisting on this one).